tests/test_pyimager.py

```diff
@@ -3,23 +3,37 @@
 import pytest
 from pyimager.pyimager import *
 
-input_path = "./tests/milad.jpg"
-def test_circropper_output():
+def test_circropper():
+    input_path = "./tests/milad.jpg"
+
+    # Test argument type 
+    try:
+        circropper(1, "one")
+        circropper(1, 10)
+        circropper(input_path, "one")
+    except TypeError:
+        pass
+
+    # Test valid image path 
+    try:
+        circropper("../../", 0)
+    except IOError:
```

Comment:
my suggested changes for the circropper function change the error type being raised. 
```suggestion
    except FileNotFoundError:
```